e2e/condition_schedule_test.go

```diff
@@ -157,6 +157,7 @@ func TestCondition_Schedule_Basic(t *testing.T) {
 			// check scheduled task did not trigger immediately and ran only on schedule
 			api.WaitForEvent(t, cts, taskName, registerTime, scheduledWait)
 			checkScheduledRun(t, taskName, registerTime, taskSchedule, port)
+			time.Sleep(defaultWaitForTestReadiness)
```

Comment:
Had to add this and the same one below this to make the E2E test pass.
Will this be because of https://github.com/hashicorp/consul-terraform-sync/pull/1067 ?

Reply:
Most likely! This is fine in the short term to get the release back on track, but could you follow up later and add a more dynamic wait? Like in `validateService`, keep checking for the expected state with more of a polling mechanism.

Reply:
Also might be better to have a shorter wait that's closer to the delay you added. So maybe 15 seconds instead of 20 seconds. But not a big deal right now since you're trying to release.